UNRELEASED.md

```diff
@@ -14,6 +14,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - Updated `Navigation` badge prop to accept a react node ([#1142](https://github.com/Shopify/polaris-react/pull/1142))
 - Changed max width on `Search` to 694px so that it is perfectly centered in the top bar ([#1107](https://github.com/Shopify/polaris-react/issues/1107))
+- Added `Empty State` footer prop ([#1200](https://github.com/Shopify/polaris-react/pull/1200))
```

Comment:
`empty state` if it's merchant facing -- otherwise fine as is.

Reply:
This is dev facing and indicates changes that are made in this PR 👍